backend/debug_r2.py

```
@@ -0,0 +1,85 @@
+import boto3
+from botocore.exceptions import ClientError
+from io import BytesIO
+from app.core.config import settings
+
+def test_r2_direct():
+    """直接测试R2操作"""
+    print("🧪 直接测试R2存储...")
+    
+    # 创建R2客户端
+    endpoint_url = f"https://{settings.R2_ACCOUNT_ID}.r2.cloudflarestorage.com"
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)